tools/pysa_integration_tests/run.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import logging
+import os
+from pathlib import Path
+
+from utils import run_pysa_integration_test, LOG
+
+
+def main() -> None:
+    """
+    Entry point function which checks if full_result.json is there, calls
+    functions from integration_test_utils to run pysa, parse
+    full_result.json, and compare the output.
+    """
+    logging.basicConfig(
+        level=logging.DEBUG,
+        format="%(asctime)s %(levelname)s %(message)s",
+    )
+
+    vulnerable_flask_app_folder = (
```

Comment:
Adding on @r0rshark's comment:
Can we make this script generic, so it can be used for both `documentation/deliberately_vulnerable_flask_app` and `stubs/integration_test`?
We could just pick up the current working directory or have an option to override it.

Reply:
Yes yes, we can do that, I was of the assumption that only `utils.py` would be reused. But, I can make the change to make `run.py` generic, that would mean less code duplication!